.github/workflows/version_bumper.yml

```diff
@@ -27,6 +27,7 @@ jobs:
     needs: discover-role-repos
     permissions:
       contents: write
+      pull-request: write
```

Comment:
Whoops, approved and merged already before I realized it's `pull-requests` and not `pull-request`